system-test/logging.ts

```diff
@@ -386,6 +412,23 @@ describe('Logging', () => {
             done();
           });
       });
+
+      it('should tail log entries as a stream', done => {
```

Comment:
I'd be interested in being able to tail entries as an async iteterator:

```
for async (const entry of this.tailEntriesAsync()) {
  console.info(entry);
}
```

Could we add a test for this? I think it makes for a much nicer interface than streams -- for folks using libraries going forward.

Reply:
sure sounds good!

Reply:
Ah I think I already have an issue tracking it here : https://github.com/googleapis/nodejs-logging/issues/988

Pls cmiiw through, I dont think `tailLogEntries` has this the async iterator feature. It's only for `listX` which is a different API endpoint than this current one I'm implementing.

Reply:
@bcoe can you let me know if I should make any other changes to this PR or give it a stamp? Thank you!